resotocore/resotocore/cli/command.py

```diff
@@ -1983,10 +1989,32 @@ def fmt_json(elem: Json) -> JsonElement:
             else:
                 return elem
 
-        def fmt(elem: JsonElement) -> JsonElement:
-            return fmt_json(elem) if isinstance(elem, dict) else str(elem)
+        async def fmt_csv(elem: Json, idx: int) -> AsyncIterator[JsonElement]:
+            def to_csv_string(lst: List[Any]) -> str:
+                output = io.StringIO()
+                dialect = csv.unix_dialect()
+                writer = csv.writer(output, dialect=dialect, quoting=csv.QUOTE_MINIMAL)
+                writer.writerow(lst)
+                return output.getvalue().rstrip()
```

Comment:
This will create a new string/writer etc for every element...
You can create this once for the whole stream and then call:
```python
output.truncate()
writer.writerrow(let)
output.getvalue()
```
for every element, reusing the buffer.